src/Bicep.Core/Semantics/NameBindingVisitor.cs

```diff
@@ -224,41 +224,6 @@ public override void VisitFunctionCallSyntax(FunctionCallSyntax syntax)
             this.bindings.Add(syntax, symbol);
         }
 
-        public override void VisitInstanceFunctionCallSyntax(InstanceFunctionCallSyntax syntax)
```

Comment:
I'll need to check whether this change breaks find-all-references between `az.someFunc()` and `someFunc()`

Reply:
I've made the fixes to support this now.